static/src/javascripts/projects/commercial/modules/messenger/get-page-targeting.ts

```diff
@@ -0,0 +1,15 @@
+import type { RegisterListener } from '../messenger';
+
+/**
+ * Register a listener for iframes to request shared ad targeting
+ *
+ * Allows for ads to be served into SafeFrame whilst retaining the ability to define a passback
+ */
+const init = (register: RegisterListener): void => {
+	register(
+		'get-page-targeting',
```

Comment:
Could we narrow the allowed string types here to a known list? Could be another PR.

Reply:
Do you the mean the strings you can register for, [like this](https://github.com/guardian/frontend/blob/8dbaec95d4fec050dc5f5023f829c98b847c26fb/static/src/javascripts/projects/commercial/modules/messenger.ts#L7-L19)?

Reply:
Ah yes, so it's already covered by the previous conversion! Thanks